noteable_magics/sql/meta_commands.py

```diff
@@ -598,7 +607,7 @@ def all_table_and_views(self, inspector) -> List[Tuple[str, str, str]]:
         default_schema = inspector.default_schema_name
         all_schemas = set(inspector.get_schema_names())
         all_schemas.difference_update(self.AVOID_SCHEMAS)
-        if default_schema not in all_schemas:
+        if default_schema and default_schema not in all_schemas:
```

Comment:
default_schema might be None, so avoid inserting it, then causing a 'Cannot compare NoneType and str' when we try to sort on line 613.